Do not use -Werror even if the compiler supports it. Newer or older
compilers than the ones the developer tested may experience
unanticipated warnings that would cause the build to fail.
--- configure.ac.orig	2021-04-02 10:51:59.000000000 -0500
+++ configure.ac	2022-04-04 17:49:07.000000000 -0500
@@ -81,7 +81,7 @@
 
 # Set some sane default CFLAGS, avoid having to do another release like 0.4.1
 if test "$ac_test_CFLAGS" != set; then
-  DEFAULT_CFLAGS="-Wall -Werror -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector --param=ssp-buffer-size=4"
+  DEFAULT_CFLAGS="-Wall -Wp,-D_FORTIFY_SOURCE=2 -fstack-protector --param=ssp-buffer-size=4"
   for F in $DEFAULT_CFLAGS; do
     AC_MSG_CHECKING([whether $CC supports $F])
     save_CFLAGS="$CFLAGS"
